dashboard/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_08_20_182004) do
+ActiveRecord::Schema.define(version: 2021_09_08_195236) do
```

Comment:
The changes to schema.rb and schema_cache.yml are not related to this change, right?

Reply:
Oops! Yes, I'll remove that, good catch. That was related to my other migration PR